include/swift/Basic/LangOptions.h

```diff
@@ -138,7 +138,7 @@ namespace swift {
 
     /// On Darwin platforms, use the pre-stable ABI's mark bit for Swift
     /// classes instead of the stable ABI's bit.
-    bool UseDarwinPreStableABIBit = !bool(SWIFT_DARWIN_ENABLE_STABLE_ABI_BIT);
+    bool UseDarwinPreStableABIBit = false;
```

Comment:
What does this guard at this point?

Reply:
The compiler still needs to emit the pre-stable ABI bit when targeting OSes prior to 10.14.4 and corresponding releases. This option controls that. It gets set based on the target OS version here: https://github.com/apple/swift/pull/23235/files#diff-487c2f92307618024a8f8f553589405dR431

Reply:
Would you mind putting a reference to that in a comment?  It seems like it would be something that could get dropped at some point (when 10.14.4 is EOL'ed).

Reply:
Sure, I'll do that. Given that we still support 10.9 that might be a while, but it would be good to explain.